src/oidc/discovery.ts

```diff
@@ -16,8 +16,14 @@ limitations under the License.
 
 import { MetadataService, OidcClientSettingsStore, SigningKey } from "oidc-client-ts";
 
-import { IDelegatedAuthConfig } from "../client";
-import { isValidatedIssuerMetadata, ValidatedIssuerMetadata, validateWellKnownAuthentication } from "./validate";
+import {
+    isValidatedIssuerMetadata,
+    ValidatedIssuerConfig,
+    ValidatedIssuerMetadata,
+    validateOIDCIssuerWellKnown,
+} from "./validate";
+import { Method, timeoutSignal } from "../http-api";
+import { AutoDiscovery } from "../autodiscovery";
 
 /**
  * @experimental
```

Comment:
this comment still talks about `m.authentication`. I think it could maybe do with some other updates too.